framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Popover/AbpPopoverTagHelperService.cs

```diff
@@ -23,6 +23,11 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
         protected virtual void SetDisabled(TagHelperContext context, TagHelperOutput output)
         {
             var triggerAsHtml = TagHelper.Dismissible ?? false ? "datatrigger=\"focus\" " : "";
+            // If not dismissable for hoverable condition
```

Comment:
![image](https://user-images.githubusercontent.com/5793291/77291281-f95f2580-6cee-11ea-98a0-cc59f87a5269.png)


They can be used together.